midterm-2023-11-06/code/5345-daniel-naumoski/index.html

```diff
@@ -9,17 +9,25 @@
 </head>
 <body>
     <h1>Bands Information</h1>
+    <div id="filter" class="filter">
+      <label for="name">Name: </label>
+      <input type="text" id="name-filter" />
```

Comment:
The `for` in the label should refer to an `id` attribute of the input, so in this case, the correct value is `name-filter`